tests/ops/graph/graph_test_util.py

```diff
@@ -95,7 +95,9 @@ def contains_keys(data: Dict[str, Any], *keys: str) -> bool:
 
     assert len(rows) >= min_size
     for row in rows:
-        assert contains_keys(row, *keys)
+        assert contains_keys(
+            row, *keys
+        ), f"assert_rows_match differs by [{','.join(set(keys).difference(set(row.keys())))}]"
```

Comment:
Added an assertion message to more easily tell which fields differ between the assertion and the actual row data

Reply:
nice! very helpful